examples/micropython/Makefile

```diff
@@ -0,0 +1,42 @@
+# name of your application
+APPLICATION = micropython
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# the MicroPython port currently only supports 32bit architectures
+BOARD_BLACKLIST := arduino-duemilanove arduino-leonardo \
+                   arduino-mega2560 arduino-nano arduino-uno chronos \
+                   jiminy-mega256rfr2 mega-xplained msb-430 msb-430h \
+                   telosb waspmote-pro wsn430-v1_3b wsn430-v1_4 z1
+
+# the MicroPython port depends on ISR_STACKSIZE, which is not available on
+# arm7, risc-v, pic32
+BOARD_BLACKLIST += avsextrem msba2
```

Comment:
I prefer using `FEATURES_BLACKLIST += arch_xx` in the package dependency directly.

Reply:
done